### ISSUE TITLE
It seems computation does not run on GPU

### ISSUE THREAD

**Issue by KleinXin** [arashno/tensorflow_multigpu_imagenet#19]
In the line 35 of run.py, it seems GPUs are not used.

![image](https://user-images.githubusercontent.com/13084622/58558604-d7724f00-8253-11e9-9031-1336f92a4702.png)

When I run the program in training mode, the memory of GPUs are not used.

I guess maybe images and labels are loaded by CPU and physical memory of the server. Only the  backpropogation is calculated on GPU.

Is that right？

Thx

**Comment by arashno** [comment]
Yes, that is right.

**Comment by encichou** [comment]
> Yes, that is right.
hello,how to calculate on GPU,change cpu to gpu?

**Comment by encichou** [comment]
hello ,my cold run slowly,your code could run fast?

**Comment by arashno** [comment]
Yes, I can process up to 250-300 images per second using a V100 GPU.

**Comment by encichou** [comment]
I use your code,but I only could process 64images per second using v100 gpu ，do you know what should I do?thanks for your reply

**Comment by arashno** [comment]
Well, it depends on many factors:
1- How many CPUs are you using?
2- How fast is your disk?
3- How much RAM you have?
4- What is your batch size? 
5- Have you resized your images to 256*256 or you resize them on the fly?

**Comment by encichou** [comment]
1. i have about 20 cpus
2.
3.RAM 300G
4.batchsize 64
5.my image shape about 256x256 .i need resize.
my main trouble is that my gpu occupancy rate is too low.many time it is 0

**Comment by arashno** [comment]
It shouldn't be 0.
Could you please share your command line?

**Comment by encichou** [comment]
> 它不应该是0. 
> 你能分享一下你的命令行吗？

![image](https://user-images.githubusercontent.com/31835913/61232288-89140380-a760-11e9-81ab-718f445405f7.png)
many parameters is your default parameters.

**Comment by arashno** [comment]
Could you set log_device_placement to True and share the device placement info?

**Comment by encichou** [comment]
> 您可以将log_device_placement设置为True并共享设备放置信息吗？

ok, 
The computer is not around me，i will  share my device placement info ,9 hours later.
